test/sys/test_socket.rs

```diff
@@ -2040,57 +2040,43 @@ pub fn test_recv_ipv6pktinfo() {
 }
 
 #[cfg(any(target_os = "android", target_os = "linux"))]
-#[cfg_attr(graviton, ignore = "Not supported by the CI environment")]
```

Comment:
If you're going to remove this line, then you should also remove the `RUSTFLAGS: --cfg graviton -D warnings` line in .cirrus.yml .